### ISSUE TITLE
Error in documentation - Using a previously loaded spaCy pipeline

### ISSUE THREAD

**Issue by rory-jardine** [microsoft/presidio#1298]
In the [Handling Text / Presidio Analyzer / Customizing the NLP Model / Spacy/Stanza](https://microsoft.github.io/presidio/analyzer/nlp_engines/spacy_stanza/) page of the documentation, the code block in the 'Using a previously loaded spaCy pipeline' has an error within it that is easily resolved. 

Error:
```python
# Create a class inheriting from SpacyNlpEngine
class LoadedSpacyNlpEngine(SpacyNlpEngine):
    def __init__(self, loaded_spacy_model):
        super.__init__()
        self.nlp = {"en": loaded_spacy_model}
```
The line `super.__init__()` is incorrect, and an error `descriptor '__init__' of 'super' object needs an argument` will flag when LoadedSpacyNlpEngine is called per
```python
# Load a model a-priori
nlp = spacy.load("en_core_web_sm")

# Pass the loaded model to the new LoadedSpacyNlpEngine
loaded_nlp_engine = LoadedSpacyNlpEngine(loaded_spacy_model = nlp)

> TypeError: descriptor '__init__' of 'super' object needs an argument
```

The fix only requires brackets for the `super().__init__()`

Fix:
```python
# Create a class inheriting from SpacyNlpEngine
class LoadedSpacyNlpEngine(SpacyNlpEngine):
    def __init__(self, loaded_spacy_model):
        super().__init__()
        self.nlp = {"en": loaded_spacy_model}
```

**Comment by omri374** [comment]
Thanks! Interested in creating a PR?

**Comment by AndreasThinks** [comment]
I've created a PR for this which I think works!

**Comment by AndreasThinks** [comment]
I think this can now be closed now that #1325 is merged?

**Comment by omri374** [comment]
Yes, thanks!
